### DISCUSSION TITLE
Cavas UI MRTK3

### DISCUSSION THREAD

**Discussion by Darkmax** [microsoft/MixedRealityToolkit-Unity#10640] [Q&A] [answered]
Hi I'm developing a new project with MRTK3 for hololens.

I'm developing my UI using canvas, beacuse I like better how to on canvas can use the tools of unity to display the information.

But I'm having some problems using theming with interface using the ui canvas, is any demo scene that is using theming with this kind of gui? all the the demos that I have scene is using the other GUI that is not using canvas.

Other question, I notice that that if I'm using UI canvas and change the backplate material to my colors all the buttons that are inside this backplates change accordonly, but I want to have custom colors with my buttons and not use the same color for all my buttons.
Can I do this with GUI canvas in MRTK3.

Thanks for your help

**Comment by Zee2** [comment, marked as answer]
Hi @Darkmax, the theming system has not yet been ported to the Canvas UI systems. We are actively working on it!
